status.go

```diff
@@ -313,6 +313,8 @@ const (
 	StatusInsufficientBalancesForStorageRent                       Status = 327
 	StatusMaxChildRecordsExceeded                                  Status = 328
 	StatusInsufficientBalancesForRenewalFees                       Status = 329
+	StatusTransactionHasUnknownFields                              Status = 330
+	StatusAccountIsImmutable                                       Status = 331
 )
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**thought:** ‏Just to mention that there will be also [ALIAS_ALREADY_ASSIGNED](https://github.com/hashgraph/hedera-protobufs/blob/717b96c1ecf3fe5f9f30fe2bcbc731cf02934aae/services/response_code.proto#L1375) status as part of HIP-583, not sure when we will need to add it to the SDK as well.